app/views/trello/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="well">
```

Comment:
I guess, you've added this just to represent if Trello works. But we'll Trello API in a bit different way. So you are free to delete all this code.